Tests/BootCamp.Chapter.Tests/PokerHandsTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using BootCamp.Chapter.Gambling;
+using BootCamp.Chapter.Gambling.Poker;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class PokerHandsTests
```

Comment:
> * I don't see any tests for `Deck`.
>   
>   * You'll need to make a couple of changes to `Deck` to fully test it. You need to be able to, from the test, control the output of the randomizer. What about passing in a mocked `IRandomizer`?
> * `Card.ToString()` needs to be tested.
> * Style nitpick: Watch unnecessary empty lines and member naming conventions. One example is `PokerComboTests` with public static members in `camelCase` instead of `PascalCase`.

You don't have to inject randomized. That was  a tricky bit :) 
- Instead of injecting randomizer, setup a system under test in a way that there is only one outcome.

Reply:
moment, do i now to make test for deck or not. The assignment says only the game directory and deck is in the gameobjects file. That is why I did not make any test for it. Deck is already tested in challenge1